### PR TITLE
Add VPG to deeprl #16

### PR DESCRIPTION
**Pull request by Het-Shah**
Removed clipping in the loss calculation, and moved VPG to deeprl.

### CODE REVIEW

**Comment by Sharad24** [comment]
Have you run this with any envs yet?

**Comment by Het-Shah** [comment]
I ran this on CartPole-v1

**Comment by Sharad24** [comment]
Ok. We can merge this once reviewed but later on could you please run for few others envs as well? 
We can add the results to `benchmark.md`.

**Comment by ajaysub110** [comment]
> I ran this on CartPole-v1

It seems to be training fine once I fix an error which arises since `env` has not been added as a param for `__init__()` in line 64 of the `policies.py` file. This is a result of one of our earlier PRs. @Het-Shah Can you fix this and push?

File: jigglypuffRL/deeprl/vpg/vpg.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            epoch_reward = 0
+            for i in range(self.actor_batch_size):
+                s = self.env.reset()
+                done = False
+                for t in range(self.timesteps_per_actorbatch):
+                    a = self.select_action(s)
+                    s, r, done, _ = self.env.step(np.array(a))
+
+                    if self.render:
+                        env.render()
```
Comment (ajaysub110) [inline jigglypuffRL/deeprl/vpg/vpg.py:186]: Should be `self.env()`

Comment (Het-Shah) [reply jigglypuffRL/deeprl/vpg/vpg.py:186]: aah sorry for this one will modify this one

**Comment by Het-Shah** [comment]
@Sharad24 Can you make the list of envs to benchmark upon in `benchmark.md`?

**Comment by Het-Shah** [comment]
@ajaysub110 updated this in both PPO and VPG, also closed the env in PPO.

**Comment by Sharad24** [comment]
Ideally we'd want to have something like [this](https://github.com/araffin/rl-baselines-zoo/blob/master/benchmark.md
). We dont really have support for Atari or PyBullet yet, so I'd say the classic gym envs like:
- Cartpole
- Acrobot
- MountainCar
etc

**Comment by ajaysub110** [comment]
Great, thanks! 

> It seems to be training fine once I fix an error which arises since `env` has not been added as a param for `__init__()` in line 64 of the `policies.py` file. This is a result of one of our earlier PRs. @Het-Shah Can you fix this and push?

Can you please fix this also?

**Comment by Het-Shah** [comment]
There is `env` parameter to `__init__()` in `MlpPolicy`. Where exactly are these changes needed? @ajaysub110

**Comment by ajaysub110** [comment]
At line 64, `super(MlpPolicy, self).__init__()` should be `super(MlpPolicy, self).__init__(env)`

**Pull request merged**